packaging/debian/copyright

```diff
@@ -0,0 +1,35 @@
+Upstream Homepage:
+    https://www.gen2vdr.de/wirbel/w_scan/index2.html
+
+Upstream Author:
+    Winfried Koehler <w_scan AT gmx-topmail DOT de>
+
+Debian Maintainers:
+    Tobias Grimm <etobi@debian.org>
+
+Copyright:
+    (c) 2006 - 2014 Winfried Koehler
```

Comment:
Update this too.

Reply:
What year to add to Copyright?
Do I need to remove old copyright years or just add anew one?

Reply:
Keep the existing copyright lines and add a new one on top: "(c) 2016 - 2020 Ștefan Talpalaru"